apps/site/scripts/prepare-blocks.ts

```diff
@@ -30,8 +30,10 @@ import {
   expandBlockMetadata,
   ExpandedBlockMetadata,
   StoredBlockInfo,
-} from "../src/lib/blocks";
-import { FRONTEND_URL } from "../src/lib/config";
+} from "../src/lib/blocks.js";
+import { FRONTEND_URL } from "../src/lib/config.js";
+
+const Ajv = _Ajv as unknown as typeof _Ajv.default;
```

Comment:
Here and below: see https://github.com/ajv-validator/ajv/issues/2132

I had to do this in a few places:

```diff
-import x from "x";
+import _x from "x";
+const x = _x as unknown as typeof _x.default;
```

This must be to do with ESM-like typings for CJS packages. If a CJS module says `export = x`, all works. If it says `export default x`, the above hack is necessary. Package patching does not work if there are more exports in addition to the default one.